app/src/fflag/cache_manager.rb

```diff
@@ -0,0 +1,27 @@
+module Fflag
+  class CacheManager
+    CACHE_KEY_PREFIX = 'fflag-cache'.freeze
+    private_constant :CACHE_KEY_PREFIX
+
+    DEFAULT_EXPIRES_IN = 15.minutes
+    private_constant :DEFAULT_EXPIRES_IN
+
+    def self.clear(identifier)
+      Rails.cache.delete(cache_key(identifier))
```

Comment:
If i'm not mistaken this is still called if the cache is not enabled.

In the case that the app doesn't have `Rails.cache` enabled it will likely blow up.

Maybe consider bypassing the Cache manager completely should the cache be enabled

Reply:
good catch!